File: ts/iq/optoutlist~get~item.ts
/** <iq type="get" xmlns="optoutlist">. Replies with one of: SuccessWithMismatch, SuccessWithMatch, InvalidRequest, InternalServerError. */
export function buildBlocklistsGetOptOutList(category?: string, dhash: string): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'optoutlist', type: 'get', category: category }, content: [
      { tag: 'item', attrs: { dhash: dhash } }, /* optional */
    ] }
}

File: go/iq/optoutlist~get~item.go
// BuildBlocklistsGetOptOutList builds <iq type="get" xmlns="optoutlist">. Replies with one of: SuccessWithMismatch, SuccessWithMatch, InvalidRequest, InternalServerError.
func BuildBlocklistsGetOptOutList(category string, dhash string) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": "s.whatsapp.net", "xmlns": "optoutlist", "type": "get", "category": category},
		Content: []waBinary.Node{
			{Tag: "item", Attrs: waBinary.Attrs{"dhash": dhash}}, // optional
		},
	}
}